vars/sendEvent.groovy

```diff
@@ -0,0 +1,9 @@
+import org.dotnet.helix.*
+
+def call(String type, Map data, Map senderMap) {
+    def sender = createSender(senderMap)
+    def eventData = new HelixEventData()
+    eventData.Type = type
+    eventData.Data = data
+    return sender.Send(eventData)
```

Comment:
Do we need error handling here? It looks like error handling is added inside here, but it rethrows the exception. Is this the desired behavior?